test/action_controller/json/include_test.rb

```diff
@@ -36,6 +36,11 @@ def render_without_include
             render json: @author, adapter: :json
           end
 
+          def render_without_associations
+            setup_data
+            render json: @author, adapter: :json, include: []
```

Comment:
There's no need anymore to go through a `render` call to test that kind of thing. Just pass the option to `SerializableResource`.